src/styles/_media.scss

```diff
@@ -0,0 +1,7 @@
+$small-desktop-width: 1024px;
```

Comment:
I would say it's tablet-width (optional)

Reply:
done